modules/codegen/src/smithy4s/codegen/internals/CodegenImpl.scala

```diff
@@ -34,7 +35,7 @@ private[codegen] object CodegenImpl { self =>
       args.specs.map(_.toIO).toSet,
       args.dependencies,
       args.repositories,
-      args.transformers,
+      withAwsTypeTransformer(args.transformers),
```

Comment:
Not sure if this is the best place to put this "default" transformer, but thought that it's a good place to start.

Open to suggestions on how to improve this.

Reply:
hmmmm so this makes sense to me because it'd also work for the CLI with no sbt/mill plugin...

but there's no way to remove this transformation, unlike if it was part of the default value:

```diff
diff --git a/modules/codegen-plugin/src/smithy4s/codegen/Smithy4sCodegenPlugin.scala b/modules/codegen-plugin/src/smithy4s/codegen/Smithy4sCodegenPlugin.scala
index 3e375184..4d496b13 100644
--- a/modules/codegen-plugin/src/smithy4s/codegen/Smithy4sCodegenPlugin.scala
+++ b/modules/codegen-plugin/src/smithy4s/codegen/Smithy4sCodegenPlugin.scala
@@ -16,6 +16,7 @@
 
 package smithy4s.codegen
 
+import smithy4s.codegen.transformers.AwsStandardTypesTransformer
 import sbt.Keys._
 import sbt.util.CacheImplicits._
 import sbt.{fileJsonFormatter => _, _}
@@ -255,7 +256,9 @@ object Smithy4sCodegenPlugin extends AutoPlugin {
     ),
     config / cleanFiles += (config / smithy4sOutputDir).value,
     config / cleanFiles += (config / smithy4sResourceDir).value,
-    config / smithy4sModelTransformers := List.empty,
+    config / smithy4sModelTransformers := List(
+      AwsStandardTypesTransformer.name
+    ),
     config / packageBin / packageOptions += {
       // This piece of logic aims at tracking the dependencies that Smithy4s used to generate
       // code at build time, in the manifest of the jar. This helps automatically pulling
diff --git a/modules/codegen/src/smithy4s/codegen/internals/CodegenImpl.scala b/modules/codegen/src/smithy4s/codegen/internals/CodegenImpl.scala
index 9358e07c..4e77ae67 100644
--- a/modules/codegen/src/smithy4s/codegen/internals/CodegenImpl.scala
+++ b/modules/codegen/src/smithy4s/codegen/internals/CodegenImpl.scala
@@ -18,7 +18,6 @@ package smithy4s.codegen
 package internals
 
 import alloy.openapi._
-import smithy4s.codegen.transformers.AwsStandardTypesTransformer
 import software.amazon.smithy.model.Model
 
 import scala.jdk.CollectionConverters._
@@ -35,7 +34,7 @@ private[codegen] object CodegenImpl { self =>
       args.specs.map(_.toIO).toSet,
       args.dependencies,
       args.repositories,
-      withAwsTypeTransformer(args.transformers),
+      args.transformers,
       args.discoverModels,
       args.localJars
     )
@@ -149,7 +148,7 @@ private[codegen] object CodegenImpl { self =>
       args.specs.map(_.toIO).toSet,
       args.dependencies,
       args.repositories,
-      withAwsTypeTransformer(args.transformers),
+      args.transformers,
       discoverModels = false,
       args.localJars
     )
@@ -157,7 +156,4 @@ private[codegen] object CodegenImpl { self =>
     Node.prettyPrintJson(ModelSerializer.builder().build.serialize(model))
   }
 
-  private def withAwsTypeTransformer(transformers: List[String]): List[String] =
-    transformers :+ AwsStandardTypesTransformer.name
-
 }

```

so originally I wanted to do that second thing, but now I don't know anymore 😬 

perhaps we can add another set of flags for removing default transformations.

Reply:
Honestly I think it's fine to hardcode it, considering what the transformation does. Let's not overthink it until somebody actually has a problem with it.